extra_data/tests/test_sourcedata.py

```diff
@@ -114,6 +114,19 @@ def test_select_trains(mock_spb_raw_run):
     assert sel.train_ids == []
     assert sel.keys() == xgm.keys()
 
+    sel = xgm[by_id[10020:10040]]
+    assert sel.train_ids == list(range(10020, 10040))
+
+    sel = xgm[by_index[:10]]
+    assert sel.train_ids == list(range(10000, 10010))
+
+    sel = xgm[10]
+    assert sel.train_ids == [10010]
```

Comment:
Okay, I see that this is the same behavior I flagged above :heavy_check_mark: so it's only a matter of documentation *if* it's indeed missing (I could have simply missed it)